plugins/ci/skills/fetch-tide-pr-status/fetch_tide_pr_status.py

```
@@ -0,0 +1,268 @@
+def clean_description(desc):
+    return re.sub(r'[\u2000-\u200f\u2028-\u202f]+', ' ', desc).strip()
+
+
+def fetch_url(url):
+    req = Request(url, headers={"User-Agent": "fetch-tide-pr-status/1.0"})
+    with urlopen(req, timeout=30) as resp:
+        return resp.read().decode()
+
+
+def gh_api(path):
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)